README.md

```diff
@@ -4,6 +4,32 @@ A collection of documents that describe best practices for Canonical web team.
 
 These can be also be viewed at https://canonical-webteam.github.io/practices/.
 
+## Run site locally
+
+This site is built using [Jekyll](https://jekyllrb.com/).
+
+To run this site locally, you'll need to first [install Ruby](https://www.ruby-lang.org/en/documentation/installation/) on your local machine.
+
+Once you have installed Ruby, you can then install [Bundler](https://bundler.io) by running;
+
+```
+gem install bundler
+```
+
+The next step is to install the Ruby gems required to run the site with Bundler;
+
+```
+bundle install
+```
+
+You can then serve the site using Jekyll;
+
+```
+jekyll serve
```

Comment:
I encountered a problem here, where `~/.gem/bin` was not in my path. Should we add a section for these things?

Reply:
Could you please add a 'Troubleshooting' section at the bottom of the README;

- Describe your problem including the error message you encountered
- Outline the steps to resolve it

Can you add this via another PR please?

Reply:
Will do :)